cli/tests/integrations/test_node.py

```diff
@@ -101,6 +101,11 @@ def test_node_ssh_slave():
     _node_ssh(['--mesos-id={}'.format(slave_id), '--master-proxy'])
 
 
+def test_node_ssh_slave_with_private_ip():
```

Comment:
we need to add skip this test on windows, see https://github.com/dcos/dcos-cli/pull/910/files#diff-67456e9c321bac44727371a2aba614dfR109